carbon-design-system/ibm-security
# chore: update to carbon 10.9.3

## Affected issues

- Resolves #339 

## Proposed changes

- updates to the latest Carbon 10.9.3 release: https://github.com/carbon-design-system/carbon/releases/tag/v10.9.3

## Testing instructions

- Review the list of changes in the release notes & ensure nothing is a breaking change for us.
- Check out our storybook to ensure that everything looks all right.
